src/Composition.tsx

```diff
@@ -0,0 +1,3 @@
+export const MyComposition = () => {
+    return null;
```

Comment:
What do you think about displaying the Zenika logo here instead of null ? It will help you and it will avoid you to do it before your PR for the video build action and also for debugging when you try to do tests on the video ? 😊

Reply:
Done ✅